### PR TITLE
Book details 1 and 2 added

### PR DESCRIPTION
**Pull request by carzoglioemanuel**
## Summary

- Completed book details 1 (description) and 2 (image).

## Screenshot

image

## Trello Card

https://trello.com/c/nq1oXtB3/26-detalle-de-libro-i-textos
https://trello.com/c/QFyh9fRX/27-detalle-de-libro-ii-imagen

### CODE REVIEW

File: scss/details.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  margin: 24px 0px 0px 24px;
+  height: 226px;
+}
+
+.book-title,
+.book-description,
+.book-title-genre,
+.book-detail {
+  font-family: 'Roboto';
+}
```
Comment (SaraIsabelSantacruz) [inline scss/details.scss]: if you going to use the same font family in all project texts, you can set it with a *

File: scss/details.scss [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  &::before {
+    content: '';
+    height: 4px;
+    display: block;
+    width: 540px;
+    background-color: $light-green;
+    position: relative;
+    top: 51px;
+  }
+}
```
Comment (SaraIsabelSantacruz) [inline scss/details.scss]: order alphabetically

**Review by SaraIsabelSantacruz — CHANGES_REQUESTED** [review]
Great Job. 👏 👏

File: scss/details.scss [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+.book-description,
+.book-title-genre,
+.book-detail {
+  font-family: 'Roboto';
+}
+
+.book-title {
+  font-size: 32px;
+  font-weight: 700;
+  &::before {
```
Comment (martinmona) [inline scss/details.scss:34]: for nested blocks add one blank line

File: scss/application.scss [hunk 4]
```diff
@@ -0,0 +1 @@
+@import 'equalizer', 'details';
```
Comment (martinmona) [inline scss/application.scss]: import google fonts

`@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');`

File: scss/details.scss [hunk 5]
```diff
@@ -0,0 +1,60 @@
+@import './variables/_colors';
+
+.book-container {
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 rgba(0, 0, 0, 0.5);
+  display: flex;
+  height: 416px;
+  width: 900px;
```
Comment (martinmona) [inline scss/details.scss:12]: add `max-width: 100%;` rule

File: scss/details.scss [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+.book-title {
+  font-size: 32px;
+  font-weight: 700;
+  &::before {
+    content: '';
+    height: 4px;
+    display: block;
+    width: 540px;
```
Comment (martinmona) [inline scss/details.scss]: same

File: .gitignore [hunk 7]
```diff
@@ -0,0 +1,10 @@
+### Sass ###
+.sass-cache/
+.scss-cache/
+*.css.map
+*.sass.map
+*.scss.map
+*.css
+
+## System ##
+.DS_Store
```
Comment (martinmona) [inline .gitignore]: Add end of file (empty line)

File: index.html [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+        <h1 class="book-title">Título del libro <span class="book-title-genre">(genero)</span></h1>
+        <p class="book-description">Autor: <span class="book-detail">Nombre del autor del libro</span></p>
+        <p class="book-description">Editorial: <span class="book-detail">Nombre de la editorial</span></p>
+        <p class="book-description">
+          Año de la publicación: <span class="book-detail">Año de la publicación</span>
+        </p>
+      </div>
+    </div>
+  </body>
+</html>
```
Comment (martinmona) [inline index.html]: Add EOF (end of file)

File: scss/details.scss [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+.book-title-genre {
+  color: $light-grey;
+  font-size: 24px;
+  font-weight: 700;
+}
+
+.book-detail {
+  color: $light-grey;
+  font-size: 20px;
+}
```
Comment (martinmona) [inline scss/details.scss]: Add end of file (empty line)

File: scss/variables/_colors.scss [hunk 10]
```diff
@@ -0,0 +1,3 @@
+$light-grey: #828282;
+$light-green: #bed23c;
+$white: #fff;
```
Comment (martinmona) [inline scss/variables/_colors.scss]: same

File: index.html [hunk 11]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+
```
Comment (martinmona) [inline index.html]: remove empty line

**Review by martinmona — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 7 above)
Comment (martinmona) [inline .gitignore]: add End of File

**Review by SaraIsabelSantacruz — APPROVED** [review]

**Review by martinmona — APPROVED** [review]

File: index.html [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="application.css" />
+  </head>
+  <body>
+    <div class="book-container">
+      <img class="book-cover" src="assets/book-cover.png" />
```
Comment (nvescio) [inline index.html]: `img` tags always have to have an `alt` attribute --> `book cover`

File: scss/details.scss [hunk 13]
```diff
@@ -0,0 +1,60 @@
...
+  display: flex;
+  height: 416px;
+  width: 900px;
+  max-width: 100%;
+}
+
+.book-cover {
+  height: 368px;
+  margin: 24px;
+  width: 261px;
```
Comment (nvescio) [inline scss/details.scss:19]: I think it's no necessary to set both `height` and `width`

**Review by nvescio — CHANGES_REQUESTED** [review]

**Review by nvescio — APPROVED** [review]

**Pull request merged**